src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2505,15 +2507,17 @@ internal static string GetConsoleWindowTitle()
         /// </exception>
         internal static void SetConsoleWindowTitle(string consoleTitle)
         {
+            if (!canSetConsoleWindowTitle)
+            {
+                // some scenarios don't allow setting the window title, so we can just ignore if this fails
+                return;
+            }
+
             bool result = NativeMethods.SetConsoleTitle(consoleTitle);
 
             if (!result)
             {
-                int err = Marshal.GetLastWin32Error();
-
-                HostException e = CreateHostException(err, "SetConsoleWindowTitle",
-                    ErrorCategory.ResourceUnavailable, ConsoleControlStrings.SetConsoleWindowTitleExceptionTemplate);
-                throw e;
+                canSetConsoleWindowTitle = false;
```

Comment:
We are totally ignoring the exception.
Should we just ignore only when the win32 error is 0x1F to handle the present problem seen?